README.md

```diff
@@ -159,6 +159,68 @@ As Android will not create our main activity before launching the app, we need t
 
 ```
 
+#### Deep Linking
+
+If you want to use deep linking with this package, you should:
+
+1. Follow [this guide](https://reactnavigation.org/docs/en/deep-linking.html).
```

Comment:
It's not really specific to React navigation 🙂

Reply:
It could be better to link to https://developer.android.com/training/app-links/deep-linking